### ISSUE TITLE
'Auto logout' or 'idle timeout' should be longer

### ISSUE THREAD

**Issue by mohankapil3** [mohankapil3/chit-chat#23]
At the moment, user gets kicked out of chat after sometime. Sometimes it happens so soon and feels like its something else, not idle timeout. This started happening more frequently since application has been deployed in AWS.

This should not happen until 15 minutes of inactivity, connection should be stable.

**Comment by mohankapil3** [comment]
Fixed by setting server heartbeat to 50 sec. Now user won't be automatically logged out after 60sec of inactivity.

Dropped idea of adding 'auto logout due to inactivity after 15 mins' as it will need considerable effort.

**Comment by mohankapil3** [comment]
STOMP 1.1 supports heartbeats. See if this could be used instead of server side heartbeat implementation, it would be optimal.

**Comment by mohankapil3** [comment]
STOMP client side heartbeats were already set (20sec). As we witnessed connection closures after 60sec even then, it proves we do need server side heartbeats. For consistency aligned both side heartbeat values to 50sec. Looks like load balancer/proxy server expects activity from server side on WebSockets to consider connection as alive. This statement is based on shallow reading on internet, won't be surprised if its wrong.

Tested with latest changes, no issues encountered, so closing this issue.
